include/sparrow/buffer.hpp

```diff
@@ -149,6 +150,10 @@ namespace sparrow
         template <class It, allocator A = allocator_type>
         constexpr buffer(It first, It last, const A& a = A());
 
+        template <class Range, allocator A = allocator_type>
```

Comment:
Consider short-cutting to `template<std::ranges::input_range Range, allocator A>`

Reply:
Done